src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -281,7 +281,7 @@ class TextToMiniTest {
       case ParseError(msg) =>
         assertEquals(
           strip("""
-            |ERROR: Line 3: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't '
+            |ERROR: Line 3: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but '\t '
```

Comment:
This error message seems misleading. `\t` could be seen as a tab character, instead of the two characters. That' why I didn't put the leading backslash..